.github/dependabot.yml

```diff
@@ -0,0 +1,98 @@
+version: 2
+updates:
+- package-ecosystem: bundler
+  directory: "/WcaOnRails"
+  schedule:
+    interval: weekly
```

Comment:
This is probably the config we want to change to `monthly` (and for npm below). WDYT?

Reply:
Yes, agreed! The previous "push" by me has just been a straight-forward rebase to re-trigger PR checks.